drivers/ukconsole/ns16550/ns16550.c

```diff
@@ -356,7 +356,7 @@ static int fdt_get_device(struct ns16550_device *dev, const void *dtb,
 
 	uk_pr_debug("ns16550 @ 0x%lx - 0x%lx\n", reg_base, reg_base + reg_size);
 
-	uk_console_init(&dev->dev, "NS16550", &ns16550_ops, 0);
+	uk_console_init(&dev->dev, "ns16550", &ns16550_ops, 0);
```

Comment:
Same here, the uppercase was correct: https://archive.org/details/bitsavers_nationaldamunicationsElementsDataBook_19316911

Reply:
This one hasn't been fixed either, unresolving here too.

Reply:
There was probably a mistake happening during push. It is back in now.